tools/playground/src/index.tsx

```diff
@@ -2,4 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { App } from './App';
 
+// eslint-disable-next-line ssr-friendly/no-dom-globals-in-module-scope
 ReactDOM.render(<App />, document.getElementById('root'));
+
+// import React from 'react';
+// import { createRoot } from 'react-dom/client';
+// import { App } from './App';
+//
+// const root = createRoot(document.getElementById('root'));
```

Comment:
React 18 update leaked here

Reply:
Yes, I did it on purpose. There is also commented code in the render site and snapshots. My goal was just to upgrade to react 18.